src/common/options/mds.yaml.in

```diff
@@ -961,6 +961,12 @@ options:
   fmt_desc: Ceph will inject MDS failure in the subtree import code
     (for developers only).
   with_legacy: true
+- name: mds_kill_dirfrag_at
```

Comment:
@batrick with the change in the config format introduced a while back, have I updated this correctly?

Reply:
add runtime flag too,  otherwise looks okay to me.